cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -102,6 +102,11 @@
         display: block;
     }
 
+    &_input {
+        // Resets for default a-text-input styles.
```

Comment:
How many styles do we have that need to come in via `a-text-input`? If we have to reset styles that may be an indication that it is a different pattern and shouldn't use a-text-input at all.